### PR TITLE
Compile and create image then push to dockerhub.

### PR DESCRIPTION
**Pull request by nicksunday**
# Resolves

CERES-355

### CODE REVIEW

File: cloudbuild.yaml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    env:
+    - 'SHORT_SHA=$SHORT_SHA'
+    - 'BRANCH_NAME=$BRANCH_NAME'
+    args:
+    - mvn
+    - compile
+    - jib:build
+    - "-B"
+    - "-Dmaven.test.skip=true"
+    - "-Djib.to.image=registry.hub.docker.com/rackspaceceres/${_CERES_APP}:${_IMAGE_TAG}"
```
Comment (itzg) [inline cloudbuild.yaml:44]: Can be `rackerlabs` org when you get a chance

File: cloudbuild.yaml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        tar -xzf /tmp/m2-cache.tar.gz
+      ) || echo 'Cache not found'
+    volumes:
+    - name: user.home
+      path: /root
+
+  - id: DOCKER_LOGIN
+    name: 'gcr.io/cloud-builders/docker'
+    entrypoint: 'bash'
+    args: ['-c', 'docker login --username=rackspaceceres --password=$$DH_PASSWORD']
```
Comment (itzg) [inline cloudbuild.yaml:30]: I'm curious what "DH" stands for?

File: pom.xml [hunk 3]
```diff
@@ -141,7 +140,7 @@
 						<image>openjdk:14</image>
 					</from>
 					<to>
-						<image>${image.prefix}${project.artifactId}</image>
+						<image>registry.hub.docker.com/${docker.image.prefix}/${project.artifactId}</image>
```
Comment (itzg) [inline pom.xml:143]: You might be able to revert this change since you ended up setting `jib.to.image` directly.

File: cloudbuild.yaml (hunk 2 above)
Comment (nicksunday) [reply cloudbuild.yaml:30]: DockerHub

Comment (itzg) [reply cloudbuild.yaml:30]: Doh, of course

File: pom.xml [hunk 4]
```diff
@@ -141,7 +140,7 @@
 						<image>openjdk:14</image>
 					</from>
 					<to>
-						<image>${image.prefix}${project.artifactId}</image>
+						<image>${image.prefix}/${project.artifactId}</image>
```
Comment (itzg) [inline pom.xml:143]: Can revert all the way to the following so that an unset `image.prefix` leaves this syntactically correct
```suggestion
						<image>${image.prefix}${project.artifactId}</image>
```

**Review by itzg — APPROVED** [review]
